gradle-plugin-examples/integ/tutorial/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+
+plugins {
+    id("java-library")
+}
+
+// The test project doesn't produce a JAR.
+tasks["jar"].enabled = false
+
+tasks.named<Test>("test") {
+    useJUnitPlatform()
+    systemProperty("buildFolder",
+        gradle.includedBuild("tutorial").projectDir.resolve("service").resolve("build").absolutePath)
+}
+
+tasks["test"].dependsOn(gradle.includedBuild("tutorial").task(":service:build"))
+
+dependencies {
+    val smithyVersion: String by project
+
+    testImplementation("software.amazon.smithy:smithy-model:$smithyVersion")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.4.0")
+    testImplementation("org.junit.jupiter:junit-jupiter-engine:5.4.0")
+    testImplementation("org.junit.jupiter:junit-jupiter-params:5.4.0")
```

Comment:
These versions are pretty old

Reply:
They are. I will update to something more recent. 

I will also set up a dependabot workflow to try to keep these gradle versions up to date. Dont want customers to be using super old versions from our templates.